### DISCUSSION TITLE
Will Fluent v9 offer any virtualization functionality?

### DISCUSSION THREAD

**Discussion by DavidIsrawi** [microsoft/fluentui#25596] [General] [answered]
Our team is in need of implementing a virtualized grid to display assets. I have researched the v8 <List/> component which offers virtualization, so I am wondering if there will be any alternative to this on v9 :)

**Comment by ling1726** [comment, marked as answer]
Hi, there might be a virtualization library shipped as a part of Fluent UI v9, However this is not finalized yet.

Fluent UI v9 components will support most open source virtualization library such as `react-window`. If you provided some more details on what you would like to virtualize we could provide some more guidance

**Reply by DavidIsrawi** [reply]
Thanks for the info.

We have an array of 100+ assets being displayed as a list. There is a menu that lets you jump to a specific category within the list, and all categories are rendered linearly. It's a single scrollable view emoji picker with Category buttons to jump around the view.
